hs_access_control/models/group.py

```diff
@@ -33,6 +33,7 @@ class GroupMembershipRequest(models.Model):
     invitation_to = models.ForeignKey(User, null=True, blank=True, related_name='iu2gmrequest')
     group_to_join = models.ForeignKey(Group, related_name='g2gmrequest')
     date_requested = models.DateTimeField(editable=False, auto_now_add=True)
+    explanation = models.TextField(null=True, blank=True)
```

Comment:
Need some tests around the model changes.

https://github.com/hydroshare/hydroshare/blob/develop/hs_access_control/tests/test_group_membership_request.py